Fixed formatting so that the sections break out

DubhAd opened the pull request:
In the numbered instructions, they became a wall of text because the (a), (b), etc sections didn't start on a fresh line

DubhAd committed:
Fixed formatting so that the sections break out

In the numbered instructions, they became a wall of text because the (a), (b), etc sections didn't start on a fresh line

tielur commented on source/_components/google_assistant.markdown:
```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
+  a. Add/Import project  
+  b. Click on `BUILD` on the `Smart home` card  
   c. Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable.
   d. Click `Done`. Then click on `Overview`, which will lead you to the app details screen.
```
New line here as well

tielur commented on source/_components/google_assistant.markdown:
```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
+1. Create a new project in the [developer console](https://console.actions.google.com/).  
+  a. Add/Import project  
+  b. Click on `BUILD` on the `Smart home` card  
   c. Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable.
```
This should read:
Click `Build your Action`, click `Add Action(s)`, click `Add your first action`, select your language

Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable. If you have set `api_password:` add this password to the URL `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant?api_password=[YOUR API PASSWORD]`)

tielur replied:
This adds more explicit instruction for adding the action, as well as mentioning that you need the `api_password` on this step too if you have one.

tielur commented on source/_components/google_assistant.markdown:
```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
 ### {% linkable_title Setup %}
 
-1. Create a new project in the [developer console](https://console.actions.google.com/).
-  a. Add/Import project
```
This should read:
Add/Import project, give it a name

tielur commented on source/_components/google_assistant.markdown:
```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
 
-1. Create a new project in the [developer console](https://console.actions.google.com/).
-  a. Add/Import project
-  b. Click on `BUILD` on the `Smart home` card
```
This should read:
Click on `Home Control` card, select `Smart home` recommendation

tielur commented on source/_components/google_assistant.markdown:
```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
+  b. Click on `BUILD` on the `Smart home` card  
   c. Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable.
   d. Click `Done`. Then click on `Overview`, which will lead you to the app details screen.
 2. You'll need to fill out most of the information on that page, but none of it really matters since you won't be addressing the App directly, only through the Smart Home functionality built into Google Assistant.
```
I dont think this entire step is needed anymore

tielur commented on source/_components/google_assistant.markdown:
```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
   c. Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable.
   d. Click `Done`. Then click on `Overview`, which will lead you to the app details screen.
 2. You'll need to fill out most of the information on that page, but none of it really matters since you won't be addressing the App directly, only through the Smart Home functionality built into Google Assistant.
-3. The final item on that page `Account linking` is required for your app to interact with Home Assistant.
```
This should read:
The final item on that page `Account linking` is required for your app to interact with Home Assistant. From the left menu under the `Advanced Options` select `Account Linking`. Select `No, I only want to allow account creation on my website`, Linking type if `OAuth`

tielur replied:
This mentions more explicitly where to setup `Account Linking`  and what `Linking type` to use.

tielur reviewed (commented):
I added some comments from my recently walk through of setting up home assistant with my new google home

DubhAd committed:
Updates based upon feedback, and testing

Reviewing the process it has changed again, so I've updated it accordingly. I'm unsure that the Simulation stage is really required any more.

tielur commented:
👍

c727 commented:
Many trailing whitespaces

DubhAd commented:
@c727 Could you highlight them then? Many of the lines have trailing whitespace to ensure that a new line starts for the lettered section, that's intentional (otherwise it looks like the current docs - a wall of text).

tielur commented:
@DubhAd I don't have write access, someone with write access has to approve via a review. It looks good to me though, and I recently setup a Google Home.

skalavala reviewed (approved):
thanks for the updates.
